godot-google-play-billing/build.gradle

```diff
@@ -2,21 +2,23 @@ plugins {
     id 'com.android.library'
 }
 
+ext.pluginVersion = "1.0.0"
+
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
 
     defaultConfig {
         minSdkVersion 18
         targetSdkVersion 29
+        versionName "$pluginVersion"
```

Comment:
@timoschwarzer you should also specify the `version` number. It's a monotonically increasing value which allows the Android system to differentiate between releases since the `versionName` as a string is meaningless for comparison.

For example, the `version` here should be `1`. For the next release, it'll be `2` regardless of the `versionName`.

Reply:
You probably meant `versionCode` :)
Added that.

Reply:
yep :D